xillio-parent/pom.xml

```diff
@@ -273,6 +273,12 @@
                 <artifactId>jackson-databind</artifactId>
                 <version>${jackson.version}</version>
             </dependency>
+            <!-- Force version of jackson-annotations as <2.9.0 does not compile with jackson-bom >2.9.0-->
```

Comment:
Maybe it's possible to fix all jackson versions at the proper ones like so:

```xml
<dependency>
            <groupId>com.fasterxml.jackson</groupId>
            <artifactId>jackson-bom</artifactId>
            <version>${jackson.version}</version>
            <scope>import</scope>
            <type>pom</type>
        </dependency>  
```

Reply:
Sadly, it is not doing the trick as it won't override the databind's dependency version of jackson-annotations.